lib/lita/services/market_manager.rb

```diff
@@ -29,7 +29,7 @@ def placed_limit_order?(user_id)
       end
 
       def add_market_order(lunch_sender_id)
-        return unless @karmanager.get_karma(lunch_sender_id).positive?
+        return unless @karmanager.get_karma(lunch_sender_id) > 0
```

Comment:
Use @karmanager.get_karma(lunch_sender_id).positive? instead of @karmanager.get_karma(lunch_sender_id) > 0.